README.md

```diff
@@ -311,17 +311,17 @@ older versions of mlpack:
 If you find a bug in mlpack or have any problems, numerous routes are available
 for help.
 
-Github is used for bug tracking, and can be found at
+Github is used for bug tracking and can be found at
 https://github.com/mlpack/mlpack/.
 It is easy to register an account and file a bug there, and the mlpack
-development team will try to quickly resolve your issue.
+development team will try to resolve your issue quickly.
 
-In addition, mailing lists are available.  The mlpack discussion list is
```

Comment:
I think this is correct as well.

Reply:
Yes, I don't see any reason to change `in addition` to `also`.